var/spack/repos/builtin/packages/fftw/package.py

```diff
@@ -39,9 +39,6 @@ class Fftw(Package):
     version('3.3.5', '6cc08a3b9c7ee06fdd5b9eb02e06f569')
     version('3.3.4', '2edab8c06b24feeb3b82bbb3ebf3e7b3')
 
-    patch('pfft-3.3.5.patch', when="@3.3.5+pfft_patches", level=0)
```

Comment:
rebase or merge, either way should work since we'll squash this PR down to one commit anyway.  @eklee15: do whichever you're more familiar with.  I tend to prefer rebasing but if you are new to it, I think resolving the conflicts can be more difficult.  If you need help with this I can maybe email you 1x1.